lib/actions/wallet.js

```diff
@@ -569,8 +574,11 @@ function sendTX(walletId, options) {
 function getMultisigProposals(walletId, options = {}) {
   return async dispatch => {
     const { multisig } = getClient();
-    const proposals = await multisig.getProposals(walletId, true);
-
+    // fetch both pending and non pending proposals
+    // and combine the two lists
```

Comment:
TODO: this will query the same proposals twice, filter out the ones that appear twice